lib/Word_Lib/Word_EqI.thy

```diff
@@ -15,8 +15,9 @@ begin
 
 text \<open>
   Some word equalities can be solved by considering the problem bitwise for all
-  @{prop "n < LENGTH('a::len)"}, which is different to running @{text word_bitwise}
-  and expanding into an explicit list of bits.
+  @{prop "n < LENGTH('a::len)"}. This is similar to to running @{text word_bitwise}
+  and expanding into an explicit list of bits, but works on words of abstract size
+  instead of concrete word sizes only.
```

Comment:
This makes it sound like it works on abstract instead of concrete-only, which is a strange way of saying it works on both?

Reply:
Hm, could be clearer, yes. What I'm trying to say is that `word_bitwise` works on concrete only, so there is a need for a proof method that can do more than that. `word_bitwise` can solve a larger set of goals, but only concrete. `word_eqI` can currently solve a smaller set of goals, but can do both abstract and concrete. Maybe I should just say that :-)

Reply:
Something like that would be clearer, yes :)